buildpipeline/pipeline.json

```diff
@@ -31,6 +31,21 @@
           }
         },
         {
+          "Name": "Core-Setup-Linux-BT",
+          "Parameters": {
+            "PB_DistroRid": "rhel.6.9-x64",
```

Comment:
I think the RID should probably be "rhel.6-x64" I don't think we have a RID for 6.9.